03_letters/undrewb/main_test.go

```diff
@@ -0,0 +1,143 @@
+package main
+
+import (
+	"bytes"
+	"reflect"
+	"strconv"
+	"strings"
+	"testing"
+)
+
+// var out io.Writer = os.Stdout
```

Comment:
commented out code should not be present in the PR

Reply:
please fix.

Reply:
fixed